templates/fort.3.mother

```diff
@@ -1,16 +1,54 @@
+GEOME-STRENG TITLE:%Runnam
+PRINTOUT OF INPUT PARAMETERS--------------------------------------------
+NEXT
+TRACKING PARAMETERS-----------------------------------------------------
+%turnss 0 %nss %ax0s %ax1s 0 %imc
+1 1 %idfor 1 %iclo6
+0 0 1 1 %writebins 50000 2
+NEXT
+INITIAL COORDINATES-----------------------------------------------------
+        2 0. 0. %ratios 0
+        0.
+        0.
+        0. 
+        0.
+        0.
+        %dp1
+        .000001 
+        0.
+        0. 
+        0.
+        0.
+        %dp2
+        %e0
+        %e0
+        %e0
+NEXT
+SYNCHROTRON OSCILLATIONS--------'PLACE AFTER TRACKING PARAMETERS'-------
+      35640 .000347 %rfvol 0. %length %pmass %ition
+      1.       1.
+NEXT
+BEAM
+%bunch_charge   %emit_beam  %emit_beam  %sigz  %sige  1  %ibtype 1 0
+
+NEXT
+ITERATION-ACCURACY------------------------------------------------------
+       50 1D-12 1D-15
+       10 1D-10 1D-10
+       10 1D-6  1D-6
+          1D-6  1D-9  1D-9
+NEXT
+
 FLUCTUATION
 100000 1 7 3
 NEXT
-%TUNETUNE VARIATION----------------------------------------------------------
-%TUNE         mq.7r3.b1..1  %tunex
-%TUNE         mq.8r3.b1..1  %tuney
-%TUNE/          0.
-%TUNE/          
-%TUNENEXT
-%CHROCHROMATICITY CORRECTION------------------------------------------------
-%CHRO         ms.11r3.b1 %chromx
-%CHRO         ms.12r3.b1 %chromy
-%CHRONEXT
+/TUNE VARIATION----------------------------------------------------------
+
+//          0.
+//          
+/NEXT
+/CHROMATICITY CORRECTION------------------------------------------------
+/NEXT
```

Comment:
concerning these lines, I would have left them as they were, and simply replace `%TUNE` and `%CHRO` with the comment character